src/w3d/renderer/texturebase.cpp

```diff
@@ -71,7 +71,7 @@ TextureBaseClass::~TextureBaseClass()
 #ifdef BUILD_WITH_D3D8
     if (m_d3dTexture != W3D_TYPE_INVALID_TEXTURE) {
         m_d3dTexture->Release();
-        m_d3dTexture = nullptr;
+        m_d3dTexture = W3D_TYPE_INVALID_TEXTURE;
```

Comment:
We use `W3D_TYPE_INVALID_TEXTURE` at some places. Might as well add more consistency here.